### DISCUSSION TITLE
No cache for orders when HPOS is enabled?

### DISCUSSION THREAD

**Discussion by Screenfeed** [woocommerce/woocommerce#36504] [WooCommerce HPOS] [answered]
Hello 👋 

While working on our compatibility in Polylang for WooCommerce, we were expecting to find a cache system for the orders when HPOS is enabled, but no luck so far.
At first we were looking for something similar to what WP does with `last_changed`, ex: `wp_cache_set( 'last_changed', microtime(), 'wc_orders' );`, but then, we couldn't find any cache at all for the orders.

So, did we miss something? Or maybe you consider that a cache is not necessary for orders?

Thanks.

**Comment by vedanshujain** [comment, marked as answer]
Hey, we had to revert it after we found a bug, we are planning to reintroduce it back with https://github.com/woocommerce/woocommerce/pull/35014 in an upcoming release.

**Comment by Screenfeed** [comment]
Thank you for your answer.
